.chloggen/sqlserver_pc_metrics.yaml

```diff
@@ -0,0 +1,33 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
I'm on the fence as to whether this should be considered an `enhancement` or `breaking` change. These metrics are all shown as being enabled by default, but the reality is they aren't received unless running on Windows.

This change does increase the number of metrics being received when directly connecting to the SQL server instance in reality, so let me know if we should mark this as `breaking` instead.

Reply:
Seems reasonable to call it an enhancement